site/data/3.12/arangod.json

```diff
@@ -10582,6 +10615,39 @@
     "section" : "rocksdb",
     "type" : "uint64"
   },
+  "rocksdb.max-write-buffer-number-zkd" : {
```

Comment:
@Simran-B this one still uses `zkd`, while the other ones have been renamed to `mdi`. Is this expected?

Reply:
Expected in the sense that the so-called external name is still "zkd" and the startup options are constructed from the column family names: https://github.com/arangodb/arangodb/blob/68ba8e98aa7d1df5a52cd9a42d6bb210becb5af2/arangod/RocksDBEngine/RocksDBOptionFeature.cpp#L1572

I hope that this can still be changed but it might not be possible because of backward compatibility. Lars is OOO at the moment, unfortunately.

Reply:
Pending PR: https://github.com/arangodb/arangodb/pull/20474